Extras/HACD/hacdHACD.h

```diff
@@ -81,38 +81,38 @@ namespace HACD
 
 		//! Gives the triangles partitionas an array of size m_nTriangles where the i-th element specifies the cluster to which belong the i-th triangle
 		//! @return triangles partition
-		const long * const							GetPartition() const { return m_partition;}
+		long *							GetPartition() const { return m_partition;}
```

Comment:
Ack, oops... I think I edited this twice (once without paying close attention, and again when I still got a warning) and accidentally removed the wrong `const` the first time. Will fix presently; thanks!